plugins/out_kinesis_firehose/firehose.c

```diff
@@ -427,6 +438,14 @@ static struct flb_config_map config_map[] = {
     "Custom endpoint for the STS API."
     },
 
+    {
+     FLB_CONFIG_MAP_STR, "compression", NULL,
+     0, FLB_FALSE, 0,
```

Comment:
Please feel free to let me know if this text should be more or less descriptive. Mostly just copied over from S3

Reply:
I am thinking about if we need to update the description. 'gzip' is not the only supported value if we enable arrow at compile time. Maybe we can make it more accurate.

Reply:
Maybe this is more accurate? How about:
```
    "Compression type for Firehose records. Each log record is individually compressed "
    "and sent to Firehose. 'gzip' and 'arrow' are the supported values. "
    "'arrow' is only an available if Apache Arrow was enabled at compile time. "
    "Defaults to no compression."
```